cmd/juju/application/deploy_test.go

```diff
@@ -1217,9 +1194,9 @@ func (s *CAASDeploySuite) TestDevices(c *gc.C) {
 
 func (s *DeploySuite) TestDeployStorageFailContainer(c *gc.C) {
 	charmDir := testcharms.RepoWithSeries("bionic").ClonedDir(c.MkDir(), "multi-series")
-	curl := charm.MustParseURL("local:focal/multi-series-1")
+	curl := charm.MustParseURL("local:jammy/multi-series-1")
```

Comment:
This change is because BaseSelector checks is latest LTS is supported before picking the charm's default